interactions/api/models/member.py

```diff
@@ -105,6 +105,16 @@ def mention(self) -> str:
         """
         return f"<@!{self.user.id}>" if self.nick else f"<@{self.user.id}>"
 
+    @property
+    def name(self) -> str:
+        """
+        Returns a string of whether the users nickname or its username, if a nickname isn't present
```

Comment:
I think this should be:
`Returns the string of either the user's nickname or username.`
The one you had was confusing and with some grammar mistakes.
BTW you were also missing a period at the end.

Reply:
I see the grammar mistake now lol

I had like 5 minutes of my computer time left and wanted to finish it :lol:

Reply:
```suggestion
        returns the string of either the user's nickname or username
```

testing if this works, I deleted the fork..